.gitignore

```diff
@@ -6,6 +6,7 @@
 /data/logs/*.log
 /data/sqlite/*.db
 /public/all.*
+/public/selfoss.appcache
```

Comment:
AppCache has been deprecated for a while now, we might want to use [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) instead.

Reply:
Service Workers do not work with safari that's why I went down this route for now.

Reply:
Also Service Workers require HTTPS.

Reply:
Safari is now supported: https://caniuse.com/#feat=serviceworkers

Regarding HTTPS, AppCache over unencrypted connection is insecure, as it allows persistent attacks. Firefox is actually [removing AppCache support in insecure contexts](https://blog.mozilla.org/security/2018/02/12/restricting-appcache-secure-contexts/) and other browsers will follow soon.